exercises/supervised-regression/ex_rnw/ex_lm_abalone.Rnw

```diff
@@ -0,0 +1,43 @@
+We want to predict the age of an abalone using its longest shell measurement and 
+its weight.
+
+See \url{http://archive.ics.uci.edu/ml/datasets/Abalone} for more details.
+
+<<echo=TRUE, message=FALSE>>=
+url <-
+  "http://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data"
+abalone <- read.table(url, sep = ",", row.names = NULL)
+colnames(abalone) <- c(
+  "sex", "longest_shell", "diameter", "height", "whole_weight", 
+  "shucked_weight", "visceral_weight", "shell_weight", "rings")
+@
+
+\begin{itemize}
+  \item[a)] Plot \texttt{longest\_shell} and \texttt{whole\_weight} on the $x$- 
+  and $y$-axis, respectively, and color points with \texttt{rings}.
+\end{itemize}
+
+Using \texttt{mlr3}:
+
+\begin{itemize}
+  \item[b)] Define a linear regression learner (for this you will need to load 
+  the \texttt{mlr3learners} extension package first) and use it to fit a 
+  linear model to the \texttt{abalone} data. For this, first create a 
+  \texttt{task} like so \textcolor{orange}{(necessary if they have heard the 
```

Comment:
nein, wird dann nicht nötig sein. Aber

- schieb die Zeile mit "data <-" nach oben in den chunk
- neue Aufgabe b in dem man nur den task definieren soll
- und dann in der neuen c (alte b) nur den learner definieren und trainieren (und: "to train" statt "to fit")